### PR TITLE
Test both the 2.0 version and the 1.x variant on travis

### PR DESCRIPTION
**Pull request by rictic**

### CODE REVIEW

**Comment by rictic** [comment]
Maybe? Or maybe we configure that at the sauce level?

What should the default browsers be? I know we're dropping IE 10, are we dropping any Safari versions?

**Comment by bicknellr** [comment]
(from offline) Sounds like we should update the browser list by adding a new list to WCT - doesn't need to be part of this PR.

**Comment by rictic** [comment]
Updating browsers in https://github.com/Polymer/wct-sauce/pull/22/files

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,8 @@
 language: node_js
 sudo: required
 before_script:
-  - npm install -g bower polylint web-component-tester eslint eslint-plugin-html
-  - bower install
-  - polylint
+  - npm install -g bower polylint web-component-tester@next eslint eslint-plugin-html polymer-cli@next
```
Comment (rictic) [inline .travis.yml]: :+1: done

**Pull request merged**